src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8503,7 +8503,7 @@ namespace SurfaceGeometry {
                     continue;
                 }
                 // Read surrounding surfaces properties
-                SrdSurfsProp.TotSurroundingSurface = NumNumeric - 1;
+                SrdSurfsProp.TotSurroundingSurface = NumNumeric - 2;
```

Comment:
I see, so in the IDD, the object has a numeric sky view factor and ground view factor, and then the list of surfaces starts with N3, so you should subtract 2.  OK, I'm good with this change by itself, it's simple, small, and obvious.